Microsoft.SystemForCrossDomainIdentityManagement/Service/ServiceConstants.cs

```diff
@@ -15,6 +15,7 @@ public static class ServiceConstants
         public const string RouteServiceConfiguration = SchemaConstants.PathInterface + ServiceConstants.SeparatorSegments + ServiceConstants.PathSegmentServiceProviderConfiguration;
         //public const string RouteToken = SchemaConstants.PathInterface + ServiceConstants.SeparatorSegments + ServiceConstants.PathSegmentToken;
```

Comment:
Please don't leave commented lines in the code. (This one was apparently left by the last programmer, so let's clean up their oversight.)